app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -104,14 +104,22 @@ def confirm_delivery
       @delivery.gogovan_order = GogovanOrder.book_order(current_user,
         order_params) if params["gogovanOrder"]
       if @delivery && @delivery.update(get_delivery_details)
-        render json: @delivery, serializer: serializer
+        render_delivery_json
       else
-        render json: @delivery.errors.to_json, status: 422
+        render_errors
       end
     end
 
     private
 
+    def render_delivery_json
+      render json: @delivery, serializer: serializer
+    end
```

Comment:
@swatijadhav I'm all for refactoring code particularly where it's re-used a few times but I think one-line methods that don't introduce any new logic aren't necessary. We're just adding an extra function call in the stack.

So for example, this method doesn't add any value. It just passes parameters onto the next function.

    def render_delivery_json
      render json: @delivery, serializer: serializer
    end

But this one does add value because it transforms the errors into a json format.

    def render_errors
      render json: @delivery.errors.to_json, status: 422
    end

Does that make sense?